### ISSUE TITLE
Mission compiler OVQT plugin crash.

### ISSUE THREAD

**Issue by ryzom-pipeline** [ryzom/ryzomcore#117]
**[Original report](https://bitbucket.org/ryzom/ryzomcore/issue/117) by Łukasz K. (Bitbucket: [maczuga](https://bitbucket.org/maczuga), GitHub: [maczuga](https://github.com/maczuga)).**

----------------------------------------

Crash happends anytime I'm trying to save any settings with this plugin on.

Crash occurs at MissionCompilerSettingsPage::writeSettings().

Lines:

```
#!c++

	for(int row = 0; row < m_ui.serversTableWidget->rowCount(); row++)
	{
		for(int column = 0; column < m_ui.serversTableWidget->columnCount(); column++)
		{
			items << m_ui.serversTableWidget->item(row, column)->text(); <-- this one.
		}
	}
```

**Comment by ryzom-pipeline** [comment]
**Original comment by Alex Roman (Bitbucket: [EIREXE](https://bitbucket.org/EIREXE), GitHub: [EIREXE](https://github.com/EIREXE)).**

----------------------------------------

What is the exact error?.

**Comment by ryzom-pipeline** [comment]
**Original comment by dfighter1985 (Bitbucket: [dfighter1985](https://bitbucket.org/dfighter1985), GitHub: [dfighter1985](https://github.com/dfighter1985)).**

----------------------------------------

Fixed this in a6c8dc4
